catroid/src/main/java/org/catrobat/catroid/stage/VisualPlacementActivity.java

```diff
@@ -173,10 +178,10 @@ protected void onCreate(@Nullable Bundle savedInstanceState) {
 
 		try {
 			String backgroundBitmapPath;
-			if (manualScreenshot.exists()) {
-				backgroundBitmapPath = manualScreenshot.getPath();
-			} else if (automaticScreenshot.exists()) {
+			if (automaticScreenshot.exists()) {
```

Comment:
switch round, see the updated description of CAT-14
"The ground image below the object should be the automatic screenshot, in case the project has been executed in the session and if the user is still in the same scene. In case the scene has not yet been executed, use the latest manual screenshot of the scene. If the scene has never been executed before, i.e., it is new, and thus no automatic or manual screenshot is available, and if the object also is not the background, then use the first look of the background of that scene as the ground image. However, if the background has no looks, no ground image shall exist and a white image is to be used for the ground. If the current object is itself the background, the ground image behind the look of the background is also white."